www/js/search.js

```diff
@@ -786,6 +789,13 @@ module.exports = {
             row = banidb.loadVerses(rowDb.VerseIDRangeStart, rowDb.VerseIDRangeEnd);
           }
           row.sessionKey = `ceremony-${ceremonyID}`;
+          row.crossPlatformID = rowDb.ID;
+          if (row.crossPlatformID === crossPlatformID || row.ID === LineID) {
```

Comment:
the logic in this map function is a bit confusing. 

you getting all the verses for the ceremony, then adjusting the lineIDs based on the crossplatformID values and if the current line is being shown? 

can you add some comments?